server/test/services/applicant/ApplicantDataTest.java

```diff
@@ -69,13 +69,30 @@ public void getApplicantName_fallsBackToWKPWhenNameIsEmail() {
   }
 
   @Test
-  public void getApplicantName_withMiddleName_exists() {
+  public void getApplicantName_withMiddleNameWithoutSuffix_exists() {
     ApplicantData data = createNewApplicantData();
     data.setUserName("First Middle Last");
     assertThat(data.getApplicantName()).isEqualTo(Optional.of("Last, First"));
     assertThat(data.getApplicant().getMiddleName().get()).isEqualTo("Middle");
   }
 
+  @Test
+  public void getApplicantName_withNameSuffixWithoutMiddleName_exist() {
+    ApplicantData data = createNewApplicantData();
+    data.setUserName("First Last Jr.");
+    assertThat(data.getApplicantName()).isEqualTo(Optional.of("Last, First"));
```

Comment:
**question:** Outside the scope of this PR, but have we confirmed whether the suffix should be included in this type of applicant name? Like should it be "Last, First, Jr."? https://english.stackexchange.com/questions/116856/where-do-you-put-the-suffix-when-listing-the-last-name-first

Reply:
Good point, I will raise it to shreya and/or sijia.